packages/web-components/src/wallet-connection/walletConnection.js

```diff
@@ -48,7 +49,6 @@ export const makeAgoricWalletConnection = async chainStorageWatcher => {
       onStatusChange({ status: 'seated', data: { txn, offerId: id } });
     } catch (e) {
       onStatusChange({ status: 'error', data: e });
-      return;
```

Comment:
Why this change? The idea here is that if the `submitSpendAction` fails, we don't want or need to watch for updates to the offer status.

Reply:
Yeah good point, got a bit too overzealous with removing stuff. have put it back